kie-dmn-feel/src/main/java/org/kie/dmn/feel/runtime/functions/BaseFEELFunction.java

```diff
@@ -77,6 +92,25 @@ public Object applyReflectively(EvaluationContext ctx, Object[] params) {
 
                 if( cm != null ) {
                     Object result = cm.apply.invoke( this, cm.actualParams );
+                    
+                    if (result instanceof Either) {
+                        Either<String, Object> either = (Either<String, Object>) result;
```

Comment:
Using the FP-Either approach enable us to concentrate the logic of the error in 1 place, here.
The specific function is just responsible to supply the error message string as Left.